src/lib/components/embla/EmblaSlide.svelte

```diff
@@ -0,0 +1,19 @@
+<li class="embla__slide web-carousel-item">
+    <div class="embla__slide__number">
+        <slot />
+    </div>
+</li>
+
+<style lang="scss">
+    @use '$scss/abstract' as *;
+
+    .embla__slide {
```

Comment:
same as here, with svelte you can just do `.slide` or even `div` or `li`

Reply:
👍🏼